src/helpers/__tests__/xhr.test.js

```diff
@@ -0,0 +1,30 @@
+import { fetch, authxhr } from '../xhr';
+
+describe('XHR helpers', () => {
+  describe('fetch', () => {
+    it('Gets data from given URL', async () => {
+      const res = await fetch(
+        'https://files.pavie.info/depot/remote/example.json'
```

Comment:
Basing tests on remote resources is not a good idea as you introduce uncontrolled side effects. You should only test your code only.

So you should create some stubs for `auth.xhr` and `crossFetch`. Jest has the right methods to do that.
The idea is to verify that `auth.xhr` and `crossFetch` are called, when it is right, and with the right arguments by using `authxhr`.

Reply:
Sure, I changed to use conventions from Jest and proper mocks. Thanks for your feedback.